### PR TITLE
For #9771, #9768: Adjust touch target for buttons in error pages.

### PR DESCRIPTION
**Pull request by mcarare**
This not modify button visible size or color.

### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/assets/shared_error_style.css [hunk 1]
```diff
@@ -64,17 +64,32 @@ p {
...
   display: block;
-  height: 36px;
+  height: 48px;
   width: 100%;
-  border-radius: 5px;
   border: 0;
+  padding: 6px 0px;
   font-family: inherit;
-  background-color: var(--primary-button-color);
+  background-color: #00000000;
```
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:72]: background color should also be updated for `.buttonSecondary`.

File: app/src/main/assets/shared_error_style.css [hunk 2]
```diff
@@ -64,17 +64,32 @@ p {
 
 button {
   display: block;
-  height: 36px;
+  height: 48px;
   width: 100%;
-  border-radius: 5px;
   border: 0;
+  padding: 6px 0px;
```
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:70]: We're combining padding and margin. We should probably just pick one so the button padding doesn't increase significantly. I think its alright to only use the padding value and remove the margin.

File: app/src/main/assets/shared_error_style.css (hunk 1 above)
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:72]: nit: 
```suggestion
  background-color: transparent;
```

**Comment by mcarare** [comment]
Closing and reopening to re-trigger Decision Task.

File: app/src/main/assets/shared_error_style.css [hunk 3]
```diff
@@ -64,17 +64,31 @@ p {
 
 button {
   display: block;
-  height: 36px;
+  height: 48px;
```
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:67]: If we want to make it clear in the code that the visible height is 36px, we could change the box sizing to be the standard element sizing.
```suggestion
  box-sizing: content-box;
  height: 36px; /* Padding gets added to this value to create actual height */
  padding: 6px 0px;
```

File: app/src/main/assets/shared_error_style.css [hunk 4]
```diff
@@ -64,17 +64,31 @@ p {
...
   color: var(--primary-button-text-color);
   font-size: 14px;
   font-weight: bold;
-  margin: var(--moz-vertical-spacing) auto;
+  margin: var(--moz-vertical-spacing) 0;
   text-align: center;
+  position: relative;
+}
+
+button:after {
```
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:81]: nit: psuedo elements usually use a double colon
```suggestion
button::after {
```

File: app/src/main/assets/shared_error_style.css [hunk 5]
```diff
@@ -90,10 +105,14 @@ hr {
 }
 
 .buttonSecondary {
-  background-color: var(--secondary-button-color);
+  background-color: transparent;
   color: var(--secondary-button-text-color);
 }
 
+.buttonSecondary {
```
Comment (NotWoods) [inline app/src/main/assets/shared_error_style.css:112]: needs the `::after` :) 

LGTM once this is fixed!
```suggestion
.buttonSecondary::after {
```

**Pull request merged**